nss-sapphire-elephants/BangazonSite
# Db product list

# Description

This branch closes https://github.com/nss-sapphire-elephants/BangazonSite/issues/1


## Type of change


- [X ] New feature (non-breaking change which adds functionality)



# Testing Instructions for Change Made

git fetch --all
git checkout db-product-list
Execute BangazonSite
Click on Product Type
You should see the category names with qty in parentheses
The first three items in each category will appear underneath the name
Clicking on the product will lead to the details
Add order button is not part of this ticket


# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] My changes generate no new warnings
- [X ] I have added test instructions that prove my fix is effective or that my feature works
